app/_src/mesh/features/meshglobalratelimit.md

```diff
@@ -412,7 +412,7 @@ spec:
 
 ### External service support
 
-To rate limit requests to [External Service](/mesh/{{page.kong_version}}/policies/external-services) you must deploy [ZoneEgress](/mesh/{{page.kong_version}}/explore/zoneegress). 
+To rate limit requests to [External Service](/mesh/{{page.kong_version}}/policies/external-services) you must deploy [ZoneEgress](/mesh/{{page.kong_version}}/production/cp-deployment/zoneegress/).
 
```

Comment:
This URL will only work for versions greater than `2.2.x`, should we add redirects to `/explore/zonegress/` for older versions?

Reply:
we could also render different links depending on the versions

Reply:
Meshglobalratelimiting only exists since 2.2.x